ci-operator/step-registry/openstack-k8s-operators/kuttl/openstack-k8s-operators-kuttl-commands.sh

```diff
@@ -92,41 +127,43 @@ if [ -f "/go/src/github.com/${ORG}/${BASE_OP}/kuttl-test.yaml" ]; then
   cd ${HOME}/install_yamls
   # Create/enable openstack namespace
   make namespace
-  # Creates storage
-  # Sometimes it fails to find container-00 inside debug pod
-  # TODO: fix issue in install_yamls
-  n=0
-  retries=3
-  while true; do
-    make crc_storage && break
-    n=$((n+1))
-    if (( n >= retries )); then
-      echo "Failed to run 'make crc_storage' target. Aborting"
-      exit 1
-    fi
+
+  storage_create
+
+  # perform a minor update if it is the openstack-operator
+  if [ ${SERVICE_NAME} == "openstack" ]; then
+    OPENSTACK_IMG_BKP=${OPENSTACK_IMG}
+
+    # deploy operators and ctlplane, to be updated
+    export OPENSTACK_IMG=${OPENSTACK_IMG_BASE_RELEASE:="quay.io/openstack-k8s-operators/openstack-operator-index:87ab1f1fa16743cad640f994f459ef14c5d2b9ca"}
```

Comment:
I have a quick question: in a kuttl job openstack operator/bundle/index is built on the fly and pushed in an rdo registry.
We could get the latest image pushed in `quay.io` and update the the latest version coming from the pending patch.
The last sha could be retrieved w/ something like:
```
> curl -s -H "Accept: application/vnd.docker.distribution.manifest.v2+json" \
    "https://quay.io/v2/openstack-k8s-operators/openstack-operator-index/manifests/latest" | jq -r '.config.digest'
```
or we could use a dedicated `tag` instead of latest to avoid updating this patch again if the digest changes.
@stuggi what do you think?

Reply:
I had expected us to use a much earlier release than this. Something like 18.0.2 (downstream)

Reply:
as said above, this is just the default. the real used index image should be configured as a follow up in https://github.com/openstack-k8s-operators/openstack-operator/blob/main/.prow_ci.env

Reply:
ok thank you for clarifying. It makes sense to pin a starting point. Maybe we can have in the future the a stable tag we can update whenever we want to move to the next FR zstream or release, and I see you have a plan for that already (based on `.prow_ci.env`).
I'm ok to keep using the `OPENSTACK_IMG` interface, I think we don't need extra parameters. Let's move forward with this then.

Reply:
got it. thanks